web/config.py

```diff
@@ -22,6 +22,14 @@
 if sys.path[0] != root:
     sys.path.insert(0, root)
 
+# The number of connections to keep open inside the connection pool.
+# A SQLITE_CONNECTION_POOL_SIZE setting of 0 indicates no limit.
```

Comment:
forgot to change the comment 😄 
\# The config database connection pool size
\# Setting this to 0 will remove any limit.

Reply:
Done.